components/dial/index.css

```diff
@@ -77,86 +113,113 @@ governing permissions and limitations under the License.
 
   vertical-align: top;
 
-  inline-size: var(--spectrum-dial-width);
-  block-size: var(--spectrum-dial-height);
-  min-block-size: var(--spectrum-dial-controls-min-height);
-  margin-inline-start: var(--spectrum-dial-controls-margin);
+  inline-size: var(--mod-dial-width, var(--spectrum-dial-width));
+  block-size: var(--mod-dial-height, var(--spectrum-dial-height));
+  min-block-size: var(--mod-dial-controls-min-height, var(--spectrum-dial-controls-min-height));
+  margin-inline-start: var(--mod-dial-controls-margin, var(--spectrum-dial-controls-margin));
 
-  border-radius: var(--spectrum-dial-border-radius);
-  position: relative;
-  display: inline-block;
+  border-radius: var(--mod-dial-border-radius, var(--spectrum-dial-border-radius));
   margin: 0;
```

Comment:
oh I see what's going on with `--spectrum-dial-controls-margin` not being applied, the `margin: 0` is over writing it now. When previously it was pulled out, see below:
``` 
[dir="ltr"] .spectrum-Dial-controls {
    margin-left: var(--spectrum-dial-controls-margin);
}
``` 

Do we need the `margin: 0`? or could we switch to `margin-block: 0`?

Reply:
I went with `margin-block: 0`